src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/GenericWhitespaceCheck.java

```diff
@@ -228,7 +231,8 @@ private void processNestedGenerics(DetailAST ast, String line, int after) {
         //                                          ^
         //   should be whitespace if followed by & -+
         //
-        final int indexOfAmp = line.indexOf('&', after);
+        final int indexOfAmp = IntStream.range(after, line.length)
```

Comment:
Please wrap on each successive method call after this line.

Reply:
done

Reply:
`.orElse(-1);` next line, as well.

Reply:
Not done, please wrap`orElse`

Reply:
Better if I do tomorrow, not in state of mind😴

Reply:
done

Reply:
Resolved.